src/components/accessibility/skip_link.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FunctionComponent } from 'react';
+import classNames from 'classnames';
+import { EuiScreenReaderOnly } from '../accessibility/screen_reader';
+
+export interface EuiSkipLinkProps {
+  /**
+   * Typically an anchor id (e.g. `#a11yMainContent`), the value provided here
+   * equates to the href for the link.
```

Comment:
```suggestion
   * equates to the href for the link
```

Just being nitpicky here for consistency. We usually on use periods in prop comments if there's more than one sentence.